chart/openfaas/templates/gateway-dep.yaml

```diff
@@ -37,20 +32,31 @@ spec:
       - name: gateway
         image: {{ .Values.gateway.image }}
         imagePullPolicy: {{ .Values.openfaasImagePullPolicy }}
+        {{- if .Values.securityContext }}
+        securityContext:
+          readOnlyRootFilesystem: true
+          runAsUser: 10001
```

Comment:
Should we add this user to faas-netes also?

Reply:
faas-netes needs to switch from cluster role to role and tested with readonly root. I prefer to do this in a different PR

Reply:
faas-netes does use a normal Role within the ./yaml/ folder - the helm files are still out of sync. Happy for this to be a follow-up PR.